bson/src/main-legacy/org/bson/BasicBSONObject.java

```diff
@@ -335,17 +335,6 @@ public BasicBSONObject append(final String key, final Object val) {
         return this;
     }
 
-    /**
-     * Returns a JSON serialization of this object
-     *
-     * @return JSON serialization
-     */
-    @Override
-    @SuppressWarnings("deprecation")
-    public String toString() {
-        return com.mongodb.util.JSONSerializers.getStrict().serialize(this);
```

Comment:
Also not accessible, nor is a codec for this class (DBObjectCodec moved to core, but there is no BSONObjectCodec).  I think this is justified though as use of this class is rare and it's only a toString method.

Reply:
Ack